src/redux/books/books.js

```diff
@@ -0,0 +1,45 @@
+// Actions
+const ADD_BOOK = 'bookstore/books/ADD';
+const REMOVE_BOOK = 'bookstore/books/REMOVE';
+
+const initialState = {
+  books: [{
+    bookId: 1,
+    Title: 'Jane Eyre',
+    Author: 'Charlotte Brontë',
+  },
+  {
+    bookId: 2,
+    Title: 'The Hunger Games',
+    Author: 'Suzanne Collins',
+  }],
+};
```

Comment:
- [ ] @elarhadu kindly ensure the initial state is an empty array this is a project requirement, since you are not implementing the redux function in your components at this stage, you can move the books to the books' components
![image](https://user-images.githubusercontent.com/61408860/215055245-4f4c0baa-8bce-40f0-8414-b1f1c6e8d717.png)